src/components/Modals/CollectionsSideBar.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="sidebar">
+    <div class="sidebar-header">
+      <div class="separator-header"></div>
+      <h1 class="top header-title">Collections</h1>
```

Comment:
There's a bit of misalignment with the Figma designs here, this text's bottom padding is a bit more than it should be (nitpicking).